Dev add menu scene feature anna

ChristianCecconi opened the pull request:
Completed a menu scene with a Quit button and 4 scenes

ChristianCecconi committed:
Add scene menu script

ChristianCecconi committed:
Add scene menu script

ChristianCecconi committed:
Add exit script

ChristianCecconi committed:
Add menu scene

LJArendse committed:
Merge branch 'main' into dev-add-menu-scene-feature-anna

LJArendse reviewed (approved):
Nice work @ChristianCecconi :+1: 
Changes look good to me :100:

LJArendse committed:
Update Menu UI/Layout

- Update title name to
  3rd AI and Games
  Summer School Game AI Jam:
  ¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯
  - CHANGE -
  Main Menu
- Update button labels
- Update button width
- Update font sizes
- Add credits banner

LJArendse committed:
Minor layout changes
